vector/src/main/java/lt/lrv/basemap/layers/Landcover.java

```diff
@@ -23,16 +23,17 @@ public class Landcover implements ForwardingProfile.FeaturePostProcessor, Forwar
 
     @Override
     public void processFeature(SourceFeature sf, FeatureCollector features) {
-        if (sf.getSource().equals(Source.GRPK) && sf.getSourceLayer().startsWith(Layer.GRPK_PLOTAI_PREFIX) && sf.canBePolygon()) {
+        if (sf.getSource().equals(Source.GRPK) && (sf.getSourceLayer().startsWith(Layer.GRPK_PLOTAI_PREFIX) || sf.getSourceLayer().startsWith(Layer.GRPK_VIETOV_P))&& sf.canBePolygon()) {
```

Comment:
There is only one VIETOV_P layer. Consider replacing `sf.getSourceLayer().startsWith(Layer.GRPK_VIETOV_P)` with sf.getSourceLayer().equals(Layer.GRPK_VIETOV_P)